crates/cast/src/cmd/erc20.rs

```diff
@@ -0,0 +1,173 @@
+use std::str::FromStr;
+
+use crate::Cast;
+use alloy_eips::BlockId;
+use alloy_ens::NameOrAddress;
+use alloy_primitives::U256;
+use clap::Parser;
+use foundry_cli::{
+    opts::RpcOpts,
+    utils::{LoadConfig, get_provider},
+};
+use foundry_common::fmt::format_uint_exp;
+use foundry_wallets::WalletOpts;
+
+#[doc(hidden)]
+pub use foundry_config::utils::*;
+
+/// ERC20 token subcommands.
+/// CLI arguments for `cast tx-pool`.
```

Comment:
this shows up in the help output, so please fix this up


```suggestion
/// Interact with ERC20 tokens.
```